agasc/agasc.py

```diff
@@ -132,6 +133,10 @@ def get_agasc_cone(ra, dec, radius=1.5, date=None, agasc_file=None,
     # Possibly expand initial radius to allow for slop due proper motion
     rad_pm = radius + (0.1 if pm_filter else 0.0)
 
+    global RA_DECS
+    if RA_DECS is None:
```

Comment:
I suppose I want to reinitialize this index if RA_DECS is None *or* if the query wants to use a different agasc_file than the last one query.  Hmph.